inc/saiport.h

```diff
@@ -384,6 +384,32 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_REMOTE_ADVERTISED_OUI_CODE,
 
+    /**
+     * @brief Enable ingress port rate
+     *
+     * Set policer id = #SAI_NULL_OBJECT_ID to disable policer on port.
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_POLICER
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_PORT_ATTR_INGRESS_RATE,
+
+    /**
+     * @brief Enable egress port rate
+     *
+     * Set policer id = #SAI_NULL_OBJECT_ID to disable policer on port.
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_POLICER
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_PORT_ATTR_EGRESS_RATE,
```

Comment:
can you move them to the end of the attributes to avoid change enum?

Reply:
Do you mean move after SAI_PORT_ATTR_SERDES_IPREDRIVER and before SAI_PORT_ATTR_END？

Reply:
The define is modified as belows. Is it ok?
 
/**
     * @brief Enable ingress port rate
     *
     * Set policer id = #SAI_NULL_OBJECT_ID to disable policer on port.
     *
     * @type sai_object_id_t
     * @flags CREATE_AND_SET
     * @objects SAI_OBJECT_TYPE_POLICER
     * @allownull true
     * @default SAI_NULL_OBJECT_ID
     */
    SAI_PORT_ATTR_INGRESS_RATE,

    /**
     * @brief Enable egress port rate
     *
     * Set policer id = #SAI_NULL_OBJECT_ID to disable policer on port.
     *
     * @type sai_object_id_t
     * @flags CREATE_AND_SET
     * @objects SAI_OBJECT_TYPE_POLICER
     * @allownull true
     * @default SAI_NULL_OBJECT_ID
     */
    SAI_PORT_ATTR_EGRESS_RATE,

    /**
     * @brief End of attributes
     */
    SAI_PORT_ATTR_END,